paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -131,6 +131,20 @@ public class CoreOptions implements Serializable {
                     .noDefaultValue()
                     .withDescription("The file path of this table in the filesystem.");
 
+    @ExcludeFromDocumentation("Internal use only")
+    public static final ConfigOption<String> WAREHOUSE_ROOT_PATH =
```

Comment:
Can you not introduce any option in this pr? Just modify data file meta.

Reply:
ok